Vignette (#17)
Pandora-IsoMemo/MapR: vignettes/how-to-use-MapR.Rmd

---
title: "how-to-use-MapR"
output: rmarkdown::html_vignette
vignette: >
  %\VignetteIndexEntry{how-to-use-MapR}
  %\VignetteEngine{knitr::rmarkdown}
  %\VignetteEncoding{UTF-8}
---

```{r, include = FALSE}
knitr::opts_chunk$set(
  collapse = TRUE,
  comment = "#>"
)
```

This vignette provides an overview on how to use MapR. MapR is an App to display temporal and temperature graphical files for Isomemo.

## Introduction

> To work with MapR, a zip file is needed, which can be uploaded after starting the app. It is also possible to select an online zip files that contains example data. The zip file must contain both the images to be displayed (stored in a folder called `data`) and a `json` file that specifies which images are displayed depending on the users selection.

Reviewer: ```suggestion
To work with MapR, a zip file is needed, which can be uploaded after starting the app. It is also possible to select an online zip file that contains example data. The zip file must contain both the images to be displayed (stored in a folder called `data`) and a `json` file that specifies which images are displayed depending on the users selection.
```

The `json` file can be an image list (`image_list.json`) that contains group-, variable- and measure names and the image belonging to each combination of these three. Also for each image, the path where the image can be found in the `data` folder, must be included.

> Alternatively, the `json` file can be a questionnaire (`questionnaire.json`). The file must contain questions that are displayed to the user. Furthermore the file must contain the images to be displayed with the information about for which answer the specific image is displayed. Also the path where the image can be found in the `data` folder, must be included again.

Reviewer: ```suggestion
Alternatively, the `json` file can be a questionnaire (`questionnaire.json`). The file must contain questions that are displayed to the user. Furthermore the file must contain the images to be displayed with the information which answer is needed to display a specific image. Also the path where the image can be found in the `data` folder, must be included again.
```
